action.yml

```diff
@@ -17,11 +17,11 @@ branding:
 runs:
   using: "composite"
   steps:
-    - run: |
-        curl https://raw.githubusercontent.com/pengx17/logseq-publish/main/Dockerfile.exec -o Dockerfile
-        docker build --build-arg VERSION=${{ inputs.version }} -t tmp .
-        container_id=$(docker create tmp)
-        docker cp $container_id:/home/logseq/graph-www ${{ inputs.dest }}
-        docker rm $container_id
-        rm -f Dockerfile
-      shell: bash
+    - uses: addnab/docker-run-action@v3
+      with:
+        image: "ghcr.io/pengx17/logseq-publish:${{ inputs.version }}"
+        options: |
+          -v ${{ github.workspace }}:/home/logseq/graph -v ${{ github.workspace }}/${{ inputs.dest }}:/home/logseq/graph-www
```

Comment:
It seems if the `dest` folder is not created yet, it will be owned by root when mount to the container, see https://github.com/moby/moby/issues/13121#issuecomment-336186993.
Can you please add another step above to create the folder as current user if it does not exist?

Reply:
Added.

Reply:
Great, thank you!